pydis_site/templates/events/pages/code-jams/the-code-style-guide.html

```diff
@@ -0,0 +1,257 @@
+    <p>Wildcard imports should be avoided in most cases. They make unclear where what comes from.</p>
+    <pre><code class="language-python"># No
+from pathlib import *
+
+# Yes
+from pathlib import Path</code></pre>
+    <p>You should use <b><a href="https://pycqa.github.io/isort/">isort</a></b> imports order specification, what means:</p>
+    <ul>
+        <li>
+            <b>Group by type:</b> order of import types should be: <code>__future__</code> imports, standard library imports,
+            third-party library imports, and finally project imports.
+        </li>
+        <li>
+            <b>Group by import method:</b> inside each group, first should come imports in format <code>import &lt;package&gt;</code>
+            and after them <code>from &lt;package&gt; import &lt;items&gt;</code>.
+        </li>
+        <li>
+            <b>Order imports alphabetically:</b> inside each import method group, imports should be ordered by package names.
+        </li>
+        <li>
+            <b>Order individual import items by type and alphabetically:</b> in <code>from &lt;package&gt; import &lt;items&gt;</code> format,
+            <code>&lt;items&gt;</code> should be ordered alphabetically, starting with bare module imports.
+        </li>
+    </ul>
+
+    <h3>Comments</h3>
+    <p>
+        Comments are really important because they help everyone understand what code does.
+        But as important as having comments is keeping them up-to-date.
+        Out-to-date, wrong comments confuse readers.
+    </p>
+    <p>Comments content should start with a capital letter and be a full sentence(s).</p>
+    <p>There are three types of comments:</p>
+    <ul>
+        <li>
+            <b>Block comments:</b> Probably most common comment type. Should be indented to the same level as the code they describe.
+            Each line in the block comment has to start with <code>#</code> what should be followed by a single space,
+            except for text indention inside the comment. To separate paragraphs, use one line containing only <code>#</code>.
+            <pre><code class="language-python">if variable is None or variable == 1:
+    # If variable is None, something went wrong previously.
+    #
+    # Here starts a new important paragraph.</code></pre>
+        </li>
+        <li>
+            <p>
+                <b>Inline comments:</b> You should prefer block comments over inline comments and use inline comments only where it is really necessary.
+                Never use inline comments to explain obvious things like what a line does.
+            </p>
+            <p>If you want to use an inline comment on a variable, think first, maybe you can use a better variable name instead.</p>
+            <p>
+                After code and before the start of inline comments should be at least two spaces.
+                Just like block comments, inline comments also have to start with <code>#</code> followed by a single space.
```

Comment:
Adding in some information about this extra requirement for inline comments.
```suggestion
                Inline comments are similar to block comments, but must include two spaces before the <code>#</code> which is then followed by a single space.
```

Reply:
The same information is at the line before this line.